WordPress/src/main/res/values/dimens.xml

```diff
@@ -70,7 +70,9 @@
     <dimen name="margin_extra_extra_small">1dp</dimen>
     <dimen name="margin_extra_small">2dp</dimen>
     <dimen name="margin_small">4dp</dimen>
+    <dimen name="margin_small_medium">6dp</dimen>
     <dimen name="margin_medium">8dp</dimen>
+    <dimen name="margin_extra_medium">10dp</dimen>
```

Comment:
Should `margin_extra_medium` be refactored to `margin_medium_large` to stay consistent with the `margin_small_medium` naming convention?